app/jobs/request_job.rb

```diff
@@ -16,24 +17,6 @@ class RequestJob < ApplicationJob
   end
 
   def perform(params = {})
-    r_body = {
-      name: "#{params[:first_name]} #{params[:last_name]}",
-      gender: 'male',
-      email: params[:email],
-      status: 'active'
-    }
-
-    req = Faraday.new do |f|
-      f.response :raise_error
-    end
-
-    response = req.post('https://gorest.co.in/public/v1/users') do |r|
-      r.params['access-token'] = '0375f841de50953f36828fd3b5d1baaacf7f7b1e788dc99cd57ea747731d39bb'
-      r.headers['Content-Type'] = 'application/json'
-      r.body = r_body.to_json
-    end
-
-    puts "status #{response.status}"
-    puts JSON.parse(response.body)
+    Api.send_data(params)
```

Comment:
What was the reason for extracting this as a module?

Reply:
Just for making it reusable and easy to test. I believe that wrapping API implementation is a good practice as it gives you plenty of reasonable benefits.